src/types/types.ts

```diff
@@ -230,6 +230,11 @@ export interface EdgeToken {
   networkLocation: JsonObject | undefined
 }
 
+export interface EdgeAsset {
+  pluginId: string
+  tokenId?: string
+}
```

Comment:
I would rather just put `pluginId` and `tokenId` right in the quote / network fee / wherever. This is how the rest of the core API does it, and so far it has felt pretty seamless.

Reply:
My original thought was to make this concept universal and replace the EdgeTokenId in the GUI since that's a misnomer. tokenId is used to refer to the single string and not the pluginId/tokenId combo.

Reply:
Yes, I agree that `EdgeTokenId` should be renamed to `EdgeAsset` in the GUI. However, that doesn't mean we need to bring that type over here.